core/test/uix/core_test.cljs

```diff
@@ -104,5 +104,31 @@
     (react-dom/unmountComponentAtNode root)
     (is (:componentWillUnmount @actual))))
 
+(deftest test-convert-props
+  (testing "shallow conversion"
+    (let [obj (attrs/convert-props
+               {:x 1
+                :y :keyword
+                :f identity
+                :style {:border :red}
```

Comment:
Does it make sense to test the kebab->camel conversion on the style map here? Maybe you can pass
```clojure
{:border :red
 :margin-top "12px"}
```